CONTRIBUTING.md

```diff
@@ -22,28 +22,27 @@ Please see [https://git.k8s.io/community/CLA.md](https://git.k8s.io/community/CL
 
 ### Pre-requisites
 
-Before you get started:
-
-- Install Go `1.12.3` or later
-- [Install Kubebuilder](https://book.kubebuilder.io/getting_started/installation_and_setup.html)
-- Kubernetes Cluster `1.13` or later (e.g. [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/))
-- [Configure kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 
+- Git
+- Go `1.12.3` or later
+- [Kubebuilder](https://book.kubebuilder.io/getting_started/installation_and_setup.html)
+- A Kubernetes Cluster running version `1.13` or later (e.g., [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/))
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
 
 ### Build Instructions
 
-- Get KUDO repo: `go get github.com/kudobuilder/kudo/`
-- `cd $GOPATH/src/github.com/kudobuilder/kudo`
+- Get the KUDO repo: `git clone https://github.com/kudobuilder/kudo.git`
+- `cd kudo`
 - `make all` to build project
-- [optionally] `make docker-build` to build docker images and `make docker-push` to push images
+- [optionally] `make docker-build` to build the Docker images
 
-When updating the structs under APIs, or any other code generated item, use `make generate` to generate the new DeepCopy structs. Use `make manifests` to generate out new YAML manifests representing these CRDs.
+When updating the structs under [APIs](https://github.com/kudobuilder/kudo/blob/master/pkg/apis/), or any other code generated item, use `make generate` to generate the new DeepCopy structs. Use `make manifests` to generate out new YAML manifests representing these CRDs.
 
 After updating CRD manifests, use `make deploy` to apply the new CRDs to your cluster.
```

Comment:
This alone isn't enough to deploy the CRDs anymore.  This just spits out the yaml to install them (and the controller).